### PR TITLE
Some more help on when to use .env files

### PR DESCRIPTION
**Pull request by lorenzo**
Added a note on when using .env files is not OK

### CODE REVIEW

**Comment by bravo-kernel** [comment]
aha 💡 👍

File: config/bootstrap.php [hunk 1]
```diff
@@ -50,7 +50,7 @@
  * variables as required.
  *
  * It is HIGHLY discouraged to use a .env file in production, due to security risks
```
Comment (bravo-kernel) [inline config/bootstrap.php]: **an** .env file ?

Comment (lorenzo) [reply config/bootstrap.php]: a `dot env`... that's at least how I read it

Comment (bravo-kernel) [reply config/bootstrap.php]: That's totally correct then and... a very interesting moment of insight into my perception of things  👍

File: composer.json [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
         "cakephp/migrations": "^2.0.0",
         "cakephp/plugin-installer": "^1.0",
-        "josegonzalez/dotenv": "3.*",
         "mobiledetect/mobiledetectlib": "2.*"
     },
     "require-dev": {
         "cakephp/bake": "^1.1",
         "cakephp/cakephp-codesniffer": "^3.0",
         "cakephp/debug_kit": "^3.15.0",
+        "josegonzalez/dotenv": "3.*",
```
Comment (ADmad) [inline composer.json]: Nice moving this to `require-dev`.

Comment (dereuromark) [reply composer.json]: Wouldnt it then also have to be in suggest for clarification?

Comment (dereuromark) [reply composer.json]: OK, I guess this is fine :)

File: config/bootstrap.php (hunk 1 above)
Comment (AD7six) [reply config/bootstrap.php]: could also have cheated `use .env fileS` :sunglasses:

Comment (bravo-kernel) [reply config/bootstrap.php]: 👏

**Pull request merged**